src/main/groovy/Foo.groovy

```diff
@@ -4,7 +4,7 @@ class Foo {
         try {
             print "An assertion in finally clause"
         } finally {
-            assert count > 2        // violation
+            assert count > 3        // violation
```

Comment:
[CodeNarc] WARNING: AssertWithinFinallyBlock: A finally block within class Foo contains an assert statement, potentially hiding the original exception, if there is one